spec/Exception/NoCoverageDriverAvailableExceptionSpec.php

```diff
@@ -14,7 +15,7 @@ class NoCoverageDriverAvailableExceptionSpec extends ObjectBehavior
 {
     public function it_is_initializable(): void
     {
-        $this->shouldBeAnInstanceOf(\RuntimeException::class);
+        $this->shouldBeAnInstanceOf(RuntimeException::class);
```

Comment:
I'd rather leave this as is and not use an extra `use` statement. Just because it is less code.

Reply:
PHPCsFixer from GrumpPHP does this.